templates/astropy.tpl

```diff
@@ -26,7 +26,47 @@
     </nav>
 
     {{ super() }}
+	
+	<!-- Author and Date Information -->
+	
+	<div class="clearfix" style='margin-top:20px;'>
+		<ul>
+			<span style="float: left;"><b>Author: </b>{author}</span> 
+			<span style="float: right;"><b>Date Created: </b> {date}</span>
+		</ul>
+	</div>
+	
+	<!-- Edit on Github link -->
```

Comment:
Similar with this - easy to miss right now sandwiched between the end of the tutorial and Disqus.

I'm thinking the section above the title should have something like:
```
Authors: name, name
Date Created: April 2014                                   Edit on Github [logo]
```